js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -322,9 +322,6 @@ const AssetGraphExplorerWithData = ({
         {
           ...explorerPath,
           opNames: [nextOpsNameSelection],
-          opsQuery: nodeIsInDisplayedGraph
-            ? explorerPath.opsQuery
-            : `${explorerPath.opsQuery},++"${token}"++`,
```

Comment:
removed this because `nodeIsInDIsplayedGraph` is guaranteed to be false here based on line 282.